src/conn/conn_sweep.c

```diff
@@ -13,6 +13,40 @@
       __wt_atomic_loadi32(&(dhandle)->session_inuse) == 0 &&      \
       __wt_atomic_load32(&(dhandle)->references) == 0)
 
+/*
+ * __sweep_check_file_handle_exists --
+ *     Check if the file dhandle exists for the table dhandle.
+ */
+static int
+__sweep_check_file_handle_exists(WT_SESSION_IMPL *session, WT_DATA_HANDLE *dhandle)
```

Comment:
I think it is doing more than simply checking whether a dhandle exists. 
Perhaps the function name and the comment should reflect that?

Reply:
@Siddhartha8899  Fixed in 4a4ca06d6d6b73a6ab2e8a04ab8b2e3570373b06